eva-accession-release/pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>eva-accession</artifactId>
+        <groupId>uk.ac.ebi.eva</groupId>
+        <version>0.2.0-SNAPSHOT</version>
```

Comment:
In fact, this should go into the 0.3.0-SNAPSHOT version, but we haven't released 0.2 yet.